core/Pos/Crypto/Aes.hs

```diff
@@ -0,0 +1,55 @@
+{-# LANGUAGE RankNTypes          #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+-- | Wrapper over AES. `encode` and `decode` use AES256 CTR mode with
+-- IV = 0.
+module Pos.Crypto.Aes
```

Comment:
I suggest to rename it to `Encryption` because we might want to have something else in the future. Also there is no way to generate random AES key. Alright for now, but this module just looks poor.